src/usr/local/www/interfaces_ppps_edit.php

```diff
@@ -294,26 +294,20 @@
 			}
 		}
 
-		// Loop through fields associated with an individual link/port and make an array of the data
-		$port_fields = array("localip", "gateway", "subnet", "bandwidth", "mtu", "mru", "mrru");
+		// Loop through each individual link/port and check max mtu
 		foreach ($_POST['interfaces'] as $iface) {
-			foreach ($port_fields as $field_label) {
-				if (isset($_POST[$field_label . $iface]) &&
-				    strlen($_POST[$field_label . $iface]) > 0) {
-					$port_data[$field_label][] = $_POST[$field_label . $iface];
-					$pconfig[$field_label][$iface] = $_POST[$field_label . $iface];
-					$parent_array = get_parent_interface($iface);
-					$parent = $parent_array[0];
-					$friendly = convert_real_interface_to_friendly_interface_name($parent);
-					if ($field_label == "mtu" && isset($config['interfaces'][$friendly]['mtu']) &&
-					    $_POST[$field_label . $iface] > ($config['interfaces'][$friendly]['mtu'] - 8)) {
-						$input_errors[] = sprintf(gettext("The MTU (%d) is too big for %s (maximum allowed with current settings: %d)."),
-						    $_POST[$field_label . $iface], $iface, $config['interfaces'][$friendly]['mtu'] - 8);
-					}
+			if (isset($_POST['mtu'][$iface]) &&
+			    strlen($_POST['mtu'][$iface]) > 0) {
+				$parent_array = get_parent_interface($iface);
+				$parent = $parent_array[0];
+				$friendly = convert_real_interface_to_friendly_interface_name($parent);
+				if (isset($config['interfaces'][$friendly]['mtu']) &&
```

Comment:
Note: This actually is better (needs to be) a !empty() rather than isset(). That has been fixed in the later PR.